Samples/Catalog/Sources/AlertController.swift

```diff
@@ -19,12 +19,94 @@
 import Foundation
 import UIKit
 
+extension UIAlertAction {
+  
+  /// Action types most commonly used
+  public enum actionType{
+    
+    ///Ok Option
+    case ok
+    
+    /// Default Cancel Option
+    case cancel
+    
+    /// Destructive action with custom title
+    case destructive(String)
+    
+    /// Custom action with title and style
+    case custom(String,UIAlertActionStyle)
```

Comment:
Nit: Space between `String` and `UIAlertActionStyle`
Also, we can use labels here... Do you think it would add the nice verbosity to it?